openmetadata-ui/src/main/resources/ui/src/locale/languages/en-us.json

```diff
@@ -494,9 +496,11 @@
     "message-plural-lowercase": "messages",
     "messaging": "Messaging",
     "messaging-lowercase": "messaging",
+    "messaging-plural": "Messagings",
     "metadata": "Metadata",
     "metadata-ingestion": "Metadata Ingestion",
     "metadata-lowercase": "metadata",
+    "metadata-plural": "Metadata",
```

Comment:
Did you forgot to pluralise?

Reply:
the word "metadata" is a plural noun, like the word "data" ;-)

Reply:
but in some languages like french, we can have one 'métadonnée' or two 'métadonnées'